### PR TITLE
Feature/news

### PR DESCRIPTION
**Pull request by rachadiannovansyah**
Overview:
- Add feature news tab status

cc: @jabardigitalservice/jds-backend 

Evidence:
project: Portal Jabar
title: Menambahkan feature tabs filter berita
participants: @rachadiannovansyah @khihadysucahyo @ayocodingit

### CODE REVIEW

File: core-service/src/domain/news.go [hunk 1]
```diff
@@ -119,6 +124,7 @@ type NewsUsecase interface {
 	Store(context.Context, *StoreNewsRequest) error
 	Update(context.Context, int64, *StoreNewsRequest) error
 	UpdateStatus(context.Context, int64, string) error
+	TabStatus(context.Context) ([]TabStatusResponse, error)
```
Comment (khihadysucahyo) [inline core-service/src/domain/news.go:127]: tab status, whyy?

Comment (khihadysucahyo) [reply core-service/src/domain/news.go:127]: beso

Comment (rachadiannovansyah) [reply core-service/src/domain/news.go:127]: hej @khihadysucahyo ! open to discussing bout it, I am wondering to use "TabStatus" or "TabStatusNews" for specific naming?
or you have any suggestions?

**Review by khihadysucahyo — APPROVED** [review]
dahlah

File: core-service/src/domain/news.go (hunk 1 above)
Comment (khihadysucahyo) [reply core-service/src/domain/news.go:127]: kalo buru2 ku approve aja dulu

**Review by ayocodingit — APPROVED** [review]

**Pull request merged**